package.json

```diff
@@ -37,33 +37,35 @@
     "format": "prettier --log-level warn --write \"./**/*.{jsx,js}\""
   },
   "dependencies": {
-    "@docusaurus/core": "^2.4.3",
-    "@docusaurus/plugin-client-redirects": "^2.4.3",
-    "@docusaurus/preset-classic": "^2.4.3",
-    "@docusaurus/theme-mermaid": "^2.4.3",
-    "@easyops-cn/docusaurus-search-local": "^0.33.6",
+    "@docusaurus/core": "^3.1.1",
```

Comment:
Docusaurus 3.2.0 is out now. I think it makes sense to change this before the merge of this PR.